bodhi/tests/client/test___init__.py

```diff
@@ -299,7 +299,7 @@ def test_severity_flag(self, send_request):
         result = runner.invoke(
             client.new,
             ['--user', 'bowlofeggs', '--password', 's3kr3t', '--autokarma', 'bodhi-2.2.4-1.el7',
-             '--severity', 'urgent'])
+             '--severity', 'urgent', '--notes', 'No description.'])
```

Comment:
I think we should test the command without --notes or --notes-file to make sure that we are getting the error message

Reply:
Thanks for pointing it out. Added a separate test for that!